static/compatibilities/ingress-nginx.yaml

```diff
@@ -1,15 +1,18 @@
 icon: https://github.com/pluralsh/plural-artifacts/blob/main/ingress-nginx/plural/icons/nginx.png?raw=true
 git_url: https://github.com/kubernetes/ingress-nginx
 release_url: https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v{vsn}
+chart_url: https://helm.nginx.com/stable
```

Comment:
this isn't the right helm repository, it's for `nginx` enterprise version of the ingress controller not `ingress-nginx` which is the kubernetes project maintained one.

Reply:
aaah, right. I developed the chart updated using `cert-manager` and was focused on fixing the `ingress-nginx` scraper when I added the URL. it should be right now.